tools/idr_download/idr_download_by_ids.py

```
@@ -203,6 +206,8 @@ def download_image_data(
                 [image_name, str(image_id)] + [str(x) for x in tile]
             )
             download_plane_as_tiff(image, tile, z_stack, channel_index, frame, fname)
+    except Exception:
+        warn('Some random error', image_warning_id)
```

This one is to capture all wired errors, that have not been caught before.
@wm75 all the other more explicit `warn()` will still be collected - nothing is overwritten. This one is only triggered if a new error that we have not seen before is raised - and finally, this one is making the tool at least useable. So we finally get 5000 images out of 200.000 ;)

---

Alas, this *does* interfere with the rest of the warnings and errors.
I've created https://gist.github.com/wm75/9580ada3926e195fa6383dd681d266b4 to demonstrate the issues.
In that gist, test 1 shows that the "random error" handling does not respect `skip_failed`.
Test 3 shows that it does swallow error messages.

---

Test 1 also reveals a malformatted warning message.